src/__experimental__/components/date-range/date-range.stories.mdx

```diff
@@ -142,6 +142,27 @@ For more information check our [Validations](?path=/docs/documentation-validatio
   </Story>
 </Preview>
 
+## Input props
```

Comment:
I would rather call this story `Allow Empty Value`

Reply:
But it is not a story for allowEmptyValue prop. 
It is different component, which does not expose this prop.
The story is named after what it does: shows an example of allowEmptyValue prop passed to appropriate component. I don't think we need to complicate things more. Especially since allowEmptyValue is just one of many props that can be passed here.

Reply:
I don't this so, we have a lot of examples of stories which have specific name because of use of some specific prop for component